lib/template/template-settings.js

```diff
@@ -25,9 +24,10 @@ const auth = {
                 return authCache[token].result
             }
         }
+        const { default: got } = await import('got')
```

Comment:
Why the async import?

Reply:
Because `import()` returns a promise

Reply:
Yes, but why import rather than require? There's a mix of commonjs and ES imports in this file

Reply:
Yes, that's the whole problem, it could be loading 2 different versions of the got library, one is CJS the other is ESM, please see the original issue #173

Reply:
Got it, I'd misunderstood the issue, 👍